package kyrinbox.api;

message GetOplogResponse {
    required string last_id = 1;
    repeated bytes log_data = 2;
}
